### PR TITLE
Enable Rector in CI and process the entire project

### PR DESCRIPTION
**Pull request by Kaspiman**

### CODE REVIEW

File: app/modules/Events/Integration/CycleOrm/EventRepository.php [hunk 1]
```diff
@@ -43,16 +43,11 @@ public function deleteAll(array $scope = []): void
             ->where($this->buildScope($scope))
             ->fetchAll();
 
-        $batchSize = 100;
-
         $batch = 0;
         foreach ($events as $event) {
             $this->em->delete($event);
-
-            if ($batch++ % $batchSize === 0) {
```
[the remark this answers is not in the document]
Comment (butschster) [reply app/modules/Events/Integration/CycleOrm/EventRepository.php:52]: Probably there was batch deletion. I want to refactor this.

File: app/src/Application/Service/HttpHandler/HandlerPipeline.php [hunk 2]
```diff
@@ -58,10 +57,6 @@ private function handlePipeline(ServerRequestInterface $request): ResponseInterf
         \assert($handler instanceof HandlerInterface);
         $this->position++;
 
-        if ($handler === null) {
```
[the remark this answers is not in the document]
Comment (butschster) [reply app/src/Application/Service/HttpHandler/HandlerPipeline.php:61]: As I see, $handler can be null, so it's better to fix assertion

File: app/modules/VarDumper/Application/Dump/HtmlDumper.php [hunk 3]
```diff
@@ -29,7 +29,7 @@ final class HtmlDumper extends CliDumper
 
     public function __construct(DumpIdGeneratorInterface $generator)
     {
-        AbstractDumper::__construct(null, null, 0);
+        AbstractDumper::__construct(null, null, 0); // CliDumper? parent??
```
[the remark this answers is not in the document]
Comment (butschster) [reply app/modules/VarDumper/Application/Dump/HtmlDumper.php:32]: No, `AbstractDumper`

**Comment by butschster** [comment]
Cool, Thx for a great work!

File: app/modules/Events/Integration/CycleOrm/EventRepository.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (Kaspiman) [reply app/modules/Events/Integration/CycleOrm/EventRepository.php:52]: `$batch++ % $batchSize === 0` is always true. It should be like: `++$batch % $batchSize === 0`

**Pull request merged**